ext/opcache/jit/zend_jit_arm64.dasc

```diff
@@ -2908,8 +2908,12 @@ static int zend_jit_trace_begin(dasm_State **Dst, uint32_t trace_num, zend_jit_t
 {
 	zend_regset regset = ZEND_REGSET_SCRATCH;
 
-	// In the x86 implementation, this clause would be conducted if ZTS is enabled or the addressing mode is 64-bit.
-	{
+#if ZTS
+	if (1) {
+#else
+	if (1) {  /* TODO: Conservative in ARM */
+	// if ((sizeof(void*) == 8 && !IS_SIGNED_32BIT(&EG(jit_trace_num)))) {
+#endif
```

Comment:
This looks wrong. We may completely remove this register-allocator check, if use only TMP registers for EG(jit_trace_num) assignment. I'll take care about this change separately.